examples/hybrid_compute/Quantinuum Hybrid Compute via pytket.ipynb

```diff
@@ -0,0 +1,887 @@
+     ]
+    }
+   },
+   "cell_type": "markdown",
+   "id": "2acd60fb",
+   "metadata": {},
+   "source": [
+    "![Quantinuum%20Logos_primary_blue_small.svg](attachment:Quantinuum%20Logos_primary_blue_small.svg)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dfb7ca44",
+   "metadata": {},
+   "source": [
+    "# Quantinuum Hybrid Compute via pytket\n",
+    "\n",
+    "This notebook contains an example of how to create and run hybrid classical-quantum workflows within the Quantinuum stack via `pytket`. \n",
+    "\n",
+    "**Note:** This notebook uses existing Wasm examples provided in this folder's subdirectories. To create and run your own Wasm functions, you'll need to set up an environment on your machine to support this workflow. Instructions for how to do this are given in this folder's README. \n",
+    "\n",
+    "* [Repeat Until Success](#repeat-until-success)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b4a3a966",
+   "metadata": {},
+   "source": [
+    "## Repeat Until Success <a class=\"anchor\" id=\"repeat-until-success\"></a>\n",
+    "\n",
+    "This is a simple example made of how you can process measurements, using classical logic via Wasm, to implement a quantum loop, or a repeat-until-success (RUS) style circuit, also called an exit-on-failure circuit.\n",
+    "\n",
+    "* [1 RUS Circuit](#1-rus-circuit)\n",
+    "* [RUS Experiment](#rus-experiment)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0496223",
+   "metadata": {},
+   "source": [
+    "### 1 RUS Circuit <a class=\"anchor\" id=\"1-rus-circuit\"></a>\n",
+    "\n",
+    "#### Import libraries\n",
+    "\n",
+    "Note the `wasm` module imported from `pytket` as well as several several conditional operators that are options. More information on the conditional operations available can be found in the user manual at [Classical and Conditional Operations](https://cqcl.github.io/pytket/manual/manual_circuit.html#classical-and-conditional-operations)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "68cd2200",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pathlib \n",
+    "\n",
+    "from pytket import wasm\n",
+    "\n",
+    "from pytket.circuit import (\n",
+    "    Circuit, BitRegister, Qubit, OpType,\n",
```

Comment:
That should be ok. I don't know if I had included those because I thought they may get used in later examples or to show what other operations are available... its ok to remove as we can add back any for later examples